pages/en/lb4/Creating-components.md

```diff
@@ -228,6 +228,66 @@ export class AuthenticationProvider {
 }
 ```
 
+## Extends Application with Mixin
+
+When binding a component to an app, you may want to extend the app with the component's 
+properties and methods. 
+This can be achieved by using mixin. 
+
+If you are not familiar with concept `mixin`, check [Mixin](Mixin.htm) to know more.
```

Comment:
`If you are not familiar with the mixin concept, see [Mixin](Mixin.htm) to learn more.`